source/software/r_package_management.rst

```diff
@@ -6,51 +6,106 @@ R package management
 Introduction
 ------------
 
-Most of the useful R packages can be installed separately. Some of those are 
-part of the centrally installed R modules. However, given the astounding number of 
-packages, it is not sustainable to install each and everyone of them system wide. 
-Fortunately, it is very easy for users to install R packages themselves.
-If you do encounter problems when doing so, do not hesitate to contact support.
+There exist thousands of R packages, available from online repositories like CRAN,
+Bioconductor or github. Depending on the R version, the more commonly used packages like `ggplot2`, `tidyverse` or `readr` 
+are either already included in the centrally installed R module or can be accessed by
+loading the R-bundle-CRAN and R-bundle-Bioconductor modules, e.g.:
+
+.. code:: r
+
+       $ module load R-bundle-Bioconductor/3.16-foss-2022b-R-4.2.2    
+
+It is possible, however, that these modules do not contain all R packages you need
+or that the package versions do not meet your requirements. In this case you will
+need to locally install those packages, as will be described below. Do not hesitate
+to contact support when encountering issues during these local installations.
 
 .. _r_package_management_standard_lib:
 
 Standard R package installation
 -------------------------------
 
-Setting up your own package repository for R is straightforward.
+Firstly, it is important to realize that R will use by default the `$VSC_HOME/R` path
+to install new packages. Since home directories have limited quota, it is not
+the recommended location to install software and e.g. `$VSC_DATA` should be used
+instead.
+
+Secondly, it should be kept in mind that R packages often include extensions written in
+compiled languages (e.g. C++ or Fortran) and that the centrally installed R modules are
+configured to compile these extensions with optimizations for the CPU architecture at hand.
+This means that such R packages cannot in general be used on different partitions than the
+one they were created on.
+
+Thirdly, R packages may also only work with certain versions of R and not with other versions.
+
+With these three considerations in mind, we recommend to use a directory structure which
```

Comment:
Your recommendation here also makes sense when the package installation happens as a job, and not on a login node! Of course using RStudio via OnDemand ensures it. using our typical `VSC_*` env vars ensures that even users who install their packages by mistake on the login node still get a unique library path for it, which would not be used when they submit a job.